tests.py

```diff
@@ -0,0 +1,117 @@
+from episode_sequence_generation import _get_episodes
+from signatures.attack_stages import MicroAttackStage
+
+
+def run_episode_test(frequencies, expected, test_name="test"):
```

Comment:
`MicroAttackStage.DATA_EXFILTRATION` and "Mamontenok" are just placeholders. They are not used, but are needed for the code to run